academic_observatory_workflows/database/schema/openalex/institutions.json

```diff
@@ -290,49 +268,49 @@
     "description": "Repositories (Sources with type: repository) that have this institution as their host_organization",
     "fields": [
       {
-        "name": "id",
+        "name": "display_name",
         "type": "STRING",
         "mode": "NULLABLE",
-        "description": "The OpenAlex ID of the repository."
+        "description": "The repositories display name."
       },
       {
-        "name": "issn_l",
+        "name": "host_organization",
         "type": "STRING",
-        "mode": "NULLABLE"
+        "mode": "NULLABLE",
+        "description": "The OpenAlex ID of the host organisation."
       },
       {
-        "name": "issn",
+        "name": "host_organization_lineage",
         "type": "STRING",
-        "mode": "REPEATED"
+        "mode": "REPEATED",
+        "description": "The host organisations lineage."
       },
       {
-        "name": "display_name",
+        "name": "host_organization_name",
```

Comment:
I think this field is "host_organization_lineage_names" and is given as a list of strings.

Reply:
This one is still a valid field: https://docs.openalex.org/api-entities/sources/source-object#host_organization_name